includes/class-wc-payments-account.php

```diff
@@ -2104,6 +2111,12 @@ public function finalize_embedded_connection( string $mode, array $additional_ar
 		$gateway->update_option( 'enabled', 'yes' );
 		$gateway->update_option( 'test_mode', 'live' !== $mode ? 'yes' : 'no' );
 
+		$capabilities = $this->onboarding_service->get_capabilities_from_request();
```

Comment:
Please add more inline docs here explaining where these capabilities come from and why we enforce them.

Reply:
Included an explanation with inline documentation.